### DISCUSSION TITLE
systemd-homed / homectl support for user creation

### DISCUSSION THREAD

**Discussion by mbgevers** [saltstack/salt#67358] [Feature Requests]
systemd-homed provides a new way of managing, roaming and encryption of Users Homedirectorys and it looks promising. At the Moment if a User is created by salt using user.present the only method of creating the user is using the "classic style" . 
Also if a user is created using homectl manualy, it is in conflict with the user.present function and the function will create the user again if it has been migrated to systemd-homed.

**Describe the solution you'd like**
Systemd-homed looks very promising in fixing many "Enterprise needs" be it Encryption or Roaming Homedirectorys and in my humble opinion it could get the Standard in the Future, i like the Classic Homedir way of working, but for certain Setups systemd-homed makes life easier and makes sense, thus providing it using Salt would be great.

**Describe alternatives you've considered**
Manual Management

**Additional context**
https://opensource.com/article/22/3/manage-users-home-directory-systemd-homed
https://wiki.archlinux.org/title/systemd-homed
https://systemd.io/HOME_DIRECTORY/

Regards
M.

**Comment by welcome** [comment]
Hi there! Welcome to the Salt Community! Thank you for making your first contribution. We have a lengthy process for issues and PRs. Someone from the Core Team will follow up as soon as possible. In the meantime, here’s some information that may help as you continue your Salt journey.
Please be sure to review our [Code of Conduct](https://github.com/saltstack/salt/blob/master/CODE_OF_CONDUCT.md). Also, check out some of our community resources including:

  - [Community Wiki](https://github.com/saltstack/community/wiki)
  - [Salt’s Contributor Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html)
  - [Join our Community Slack](https://via.vmw.com/salt-slack)
  - [IRC on LiberaChat](https://web.libera.chat/#salt)
  - [Salt Project YouTube channel](https://www.youtube.com/channel/UCpveTIucFx9ljGelW63-BWg)
  - [Salt Project Twitch channel](https://www.twitch.tv/saltprojectoss)

There are lots of ways to get involved in our community. Every month, there are around a dozen opportunities to meet with other contributors and the Salt Core team and collaborate in real time. The best way to keep track is by subscribing to the Salt Community Events Calendar.
If you have additional questions, email us at saltproject@vmware.com. We’re glad you’ve joined our community and look forward to doing awesome things with you!
